Frontend/ui-customer/src/Services/API.purs

```diff
@@ -3802,3 +3802,31 @@ instance standardEncodeGetManuallySharedDetailsRes :: StandardEncode GetManually
 instance showGetManuallySharedDetailsRes :: Show GetManuallySharedDetailsRes where show = genericShow
 instance decodeGetManuallySharedDetailsRes :: Decode GetManuallySharedDetailsRes where decode = defaultDecode
 instance encodeGetManuallySharedDetailsRes :: Encode GetManuallySharedDetailsRes where encode = defaultEncode
+
+
+
+-------------------------- CallOnClick API ---------------------------------------------------------------------------
+
+data CallOnClickReq = CallOnClickReq String
+
+newtype CallOnClickResp = CallOnClickResp
+  {
+    result :: String
+  }
```

Comment:
we can use the existing `result` response and its instance for this API as used at other places, similarly for driver